ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/ApiConfiguration.java

```diff
@@ -18,64 +18,126 @@
 
 import org.immutables.value.Value;
 
+/** The type Api configuration. */
```

Comment:
Wondering if this comment is configurable with your tool? It's turning the class name into a sentence which is a potentially a little confusing, e.g. there's no "Api" type here